### ISSUE TITLE
utf8_encode() messing up foreign characters

### ISSUE THREAD

**Issue by Jolly-Pirate** [nZEDb/nZEDb#2424]
###Expected behaviour
Proper UTF-8, especially for foreign characters (french, german, etc)

###Actual behaviour
The PHP function utf8_encode is messing up the foreign characters in the mysql database. 
For example, **cachée** shows up as **cachÃ©e**

###Steps to reproduce the behaviour
Try to add and backfill alt.binaries.divx.french as an example (not too big, about 4 million articles)

###Solution
Remove utf8_encode(), it's not needed anyways because PDO charset is already set to utf-8, so it's writing to the database properly. I removed it from the following files: nzb-import.php, Binaries.php, CollectionsCleaning.php, NZBImport.php, ReleaseFiles.php, ReleaseCleaning.php.

As a php test, try the following:
echo utf8_encode('cachée'); // shows messed up characters
echo utf8_decode(utf8_encode('cachée')) // reverse the process, good characters

utf8_encode() was addedn nZEDb in 2013, according to Changelog!

And remember, **utf8_encode — Encodes an ISO-8859-1 string to UTF-8**
and not all strings are ISO-8859-1. In fact, usenet articles are in UTF-8 already.

**Comment by niel** [comment]
We've wrestled with this problem. Usenet articles are not UTF-8 by default. They are not guaranteed to be UTF-8 by the RFC, which leaves us with a problem on how to handle them.

**Comment by Jolly-Pirate** [comment]
Indeed, I struggled myself with UTF-8 quite often in the past. I even had to edit the php files in utf8 (with Netbeans) to make things work properly! But once the mysql connection, html page, mail server and php files are all in utf8, the elements start to work in harmony.
I found this nice solution, you might want to add Encoding::fixUTF8() after you've done utf8_encode():
http://stackoverflow.com/questions/910793/detect-encoding-and-make-everything-utf-8
